nodejs/stream/BidirectionalStream.ts

```diff
@@ -91,6 +93,13 @@ export class BidirectionalStream {
         this._responseCollector.close(error);
         this._responsePartCollector.close(error);
         this._dispatcher.close();
+        for (const callback of this._onClose) {
+            callback(error);
```

Comment:
I think these should be `async` - and therefore `await` here and take in async callbacks